test/_data/directives_ordering/sort_directive_sections_alphabetically/bad.dart

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2017, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:async'; // OK
+import 'dart:html'; // OK
+import 'dart:convert'; // LINT
+import 'dart:math'; // OK
+
+import 'package:ansicolor/ansicolor.dart'; // OK
+import 'package:charcode/ascii.dart'; // OK
+import 'package:async/async.dart'; // LINT
+import 'package:yaml/yaml.dart'; // OK
+
+import 'package:linter/src/analyzer.dart'; // OK
+import 'package:linter/src/formatter.dart'; // OK
+import 'package:linter/src/ast.dart'; // LINT
+import 'package:linter/src/rules.dart'; // OK
+
+import 'dummy1.dart'; // OK
+import 'dummy3.dart'; // OK
+import 'dummy2.dart'; // LINT
+import 'dummy4.dart'; // OK
+
+export 'dummy1.dart'; // OK
+export 'dummy3.dart'; // OK
+export 'dummy2.dart'; // LINT
```

Comment:
Ok, but the rule says that a node is in order if its uriContent is higher alphabetically than each node before it in the group, so it can't be a not-in-order node at the beginning of a group.

Example:

```
import 'z/dart'; // OK because is higher than all nodes before it (there aren't nodes before this)
import 'a/dart'; // LINT because is not higher than z.dart
import 'b/dart'; // LINT because is not higher than z.dart
import 'c/dart'; // LINT because is not higher than z.dart
```

I will test more cases